litani.nix

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, targets ? [ ]
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "litani";
+  version = "8002c240ef4f424039ed3cc32e076c0234d01768";
+  src = fetchFromGitHub {
+    owner = "awslabs";
+    repo = "aws-build-accumulator";
+    rev = "8002c240ef4f424039ed3cc32e076c0234d01768";
+    sha256 = "sha256-UwF/B6lpsjpQn8SW+tCfOXTp14pNBr2sRGujJH3iPLk=";
+  };
+  dontConfigure = true;
+  installPhase = ''
+    mkdir -p $out/litani/
+    cp  litani $out/litani/
+    cp -r lib $out/litani/
+    cp -r templates $out/litani/
```

Comment:
Not sure why symblink is needed here, I would suggest to do this instead, maybe we can get rid of unnecessary symblink
```
mkdir -p $out/bin
cp litani $out/bin
cp -r lib $out/
cp -r templates $out/
```